src/utils/exec.ts

```diff
@@ -32,16 +32,19 @@ export async function exec(dir: string, cliArgs: string): Promise<string> {
 }
 
 /**
- * On macOS, we need to fix the $PATH environment variable
+ * On macOS & Linux, we need to fix the $PATH environment variable
  * so that we can call `npm`.
  *
  * @returns {Promise<void>}
  */
 export async function maybeFixPath(): Promise<void> {
-  if (!_fixPathCalled && process.platform !== 'win32') {
-    const fixPaths = require('fix-path');
-    fixPaths();
+  if (!_shellPathCalled && process.platform !== 'win32') {
+    const shellPaths = require('shell-path');
+    const paths = await shellPaths();
+    if (paths) {
+      process.env.PATH = paths;
```

Comment:
Is overriding the existing `PATH` intentional here (as opposed to appending to it)? I see `fixPaths` did that anyways in its source code, though.

Reply:
Yes, it's intentional.